content/posts/continuous_training/building_training_pipeline.md

```diff
@@ -106,7 +157,7 @@ The model is now ready to be used for inference. To test it on an image, simply
 poetry run python -m animal_classifier inference 1.png base_model.pth --frames-dir data/cats_and_dogs/frames --model-dir models/cats_and_dogs --threshold 0.5
```

Comment:
When you say "make sure you have poetry installed", i would add a link to poetry installation website because you describe what react is what nodejs but you assume poetry is known  (it is way less popular)